data/tlg0011/tlg003/tlg0011.tlg003.perseus-grc2.xml

```diff
@@ -29,16 +28,22 @@
             <biblStruct>
                <monogr>
                   <author>Sophocles</author>
-                  <title>Sophocles. Vol 2: Ajax. Electra. Trachiniae. Philoctetes With an
-English translation by F. Storr. The Loeb classical library, 21.
-</title>
+                  <title>Sophocles</title>
+                  <title type="sub">Ajax. Electra. Trachiniae. Philoctetes</title>
                   <editor>Francis Storr</editor>
                   <imprint>
-                     <pubPlace>London; New York</pubPlace>
-                     <publisher>William Heinemann Ltd.; The Macmillan Company</publisher>
+                     <pubPlace>London</pubPlace>
+                     <pubPlace>New York</pubPlace>
+                     <publisher>William Heinemann Ltd.</publisher>
+                     <publisher>The Macmillan Company</publisher>
                      <date>1913</date>
                   </imprint>
+                  <biblScope unit="volume">2</biblScope>
                </monogr>
+               <series xml:lang="lat">
```

Comment:
This series got tagged as Latin throughout?

Reply:
Duh, how did I miss that?